<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screenshots</class>
 <widget class="QMainWindow" name="Screenshots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>798</width>
         <height>575</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ImageLabel" name="picLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="createShot">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="createShotFull"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="delayMenu">
    <property name="title">
     <string>延迟(&amp;D)</string>
    </property>
    <addaction name="delay0"/>
    <addaction name="delay1"/>
    <addaction name="delay2"/>
    <addaction name="delay3"/>
    <addaction name="delay4"/>
    <addaction name="delay5"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="cutAction"/>
   </widget>
   <addaction name="createShot"/>
   <addaction name="menu_E"/>
   <addaction name="delayMenu"/>
  </widget>
  <action name="action_S">
   <property name="text">
    <string>全屏截图(&amp;S)</string>
   </property>
  </action>
  <action name="delay0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>无延迟</string>
   </property>
  </action>
  <action name="delay1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 秒</string>
   </property>
  </action>
  <action name="delay2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 秒</string>
   </property>
  </action>
  <action name="delay3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 秒</string>
   </property>
  </action>
  <action name="delay4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4 秒</string>
   </property>
  </action>
  <action name="delay5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 秒</string>
   </property>
  </action>
  <action name="createShotFull">
   <property name="text">
    <string>截取全屏(&amp;N)</string>
   </property>
  </action>
  <action name="actionaaa">
   <property name="text">
    <string>aaa</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>裁剪(&amp;C)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>imagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
